pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -651,6 +651,175 @@ var _ = Describe("Snapshotter", func() {
 					})
 				})
 			})
+			Describe("###GarbageCollectChunkSnapshots", func() {
+				const (
+					testDir = "garbagecollector_chunksnapshots.bkp"
+				)
+
+				var snapshotterConfig *brtypes.SnapshotterConfig
+
+				BeforeEach(func() {
+					snapshotterConfig = &brtypes.SnapshotterConfig{
+						FullSnapshotSchedule:     schedule,
+						DeltaSnapshotPeriod:      wrappers.Duration{Duration: 10 * time.Minute},
+						DeltaSnapshotMemoryLimit: brtypes.DefaultDeltaSnapMemoryLimit,
+						GarbageCollectionPeriod:  wrappers.Duration{Duration: garbageCollectionPeriod},
+						GarbageCollectionPolicy:  brtypes.GarbageCollectionPolicyLimitBased,
+						MaxBackups:               maxBackups,
+					}
+				})
+
+				AfterEach(func() {
+					err = os.RemoveAll(path.Join(outputDir, testDir))
+					Expect(err).ShouldNot(HaveOccurred())
+				})
+				Context("Fresh cluster with no chunks or snapshots", func() {
+					It("cannot delete any chunks", func() {
+						store, ssr := getStoreAndSnapshotterFromConfigs(testDir, snapshotterConfig, etcdConnectionConfig, compressionConfig, healthConfig, snapstoreConfig)
```

Comment:
Can we move store creation part to `BeforeEach`, since it is reparative in following test cases

Reply:
Yeah makes sense 👍